runtime/libffi/z/ffi64.c

```diff
@@ -875,6 +921,8 @@ ffi_prep_cif_machdep(ffi_cif *cif)
 	{
 		type = ffi_check_struct_type (*ptr);
 
+      (*ptr)->type = ffi_check_struct_for_complex(*ptr);
```

Comment:
Please use the same indentation here as line 922 (two tabs).

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/compare/583867825be4477a8319f21ffb952780631fbeb3..f6eb07952b7a7b7c8c8789fbb19e2c66f7672683